Objects/longobject.c

```diff
@@ -42,6 +42,8 @@ PyObject *_PyLong_One = NULL;
 */
 static PyLongObject small_ints[NSMALLNEGINTS + NSMALLPOSINTS];
 
+/* If use inline functions here, may lose performance due to
+   unnecessary type casting. */
 #define IS_SMALL_INT(ival) (-NSMALLNEGINTS <= (ival) && (ival) < NSMALLPOSINTS)
```

Comment:
Address STINNER Victor's advice: 
> Morever, if using a static inline function is causing issues, it would be nice to add a comment to explain why, so the issue will be avoided in the future.
https://bugs.python.org/issue37812#msg352670